kedro/framework/cli/micropkg.py

```diff
@@ -0,0 +1,293 @@
+@env_option(
+    help="Environment where the micro-package configuration lives. Defaults to `base`."
+)
+@click.option(
+    "--alias",
+    type=str,
+    default="",
+    callback=_check_pipeline_name,
+    help="Alternative name to package under.",
+)
+@click.option(
+    "-d",
+    "--destination",
+    type=click.Path(resolve_path=True, file_okay=False),
+    help="Location where to create the wheel file. Defaults to `dist/`.",
+)
+@click.option(
+    "-v",
+    "--version",
+    type=str,
+    help="Version to package under. "
+    "Defaults to micro-package package version or, "
+    "if that is not defined, the project package version.",
+)
+@click.option(
+    "--all",
+    "-a",
+    "all_flag",
+    is_flag=True,
+    help="Package all micro-packages in the `pyproject.toml` package manifest section.",
+)
+@click.argument("name", nargs=1, required=False, callback=_check_module_path)
+@click.pass_obj  # this will pass the metadata as first argument
+def package_pipeline(
+    metadata: ProjectMetadata, name, env, alias, destination, version, all_flag
+):  # pylint: disable=too-many-arguments
+    """Package up a modular pipeline or micro-package as a Python .whl."""
+    if not name and not all_flag:
+        click.secho(
+            "Please specify a micro-package name or add '--all' to package all micro-packages in "
+            "the `pyproject.toml` package manifest section."
+        )
+        sys.exit(1)
+
+    if all_flag:
+        _package_pipelines_from_manifest(metadata)
+        return
+
+    result_path = _package_pipeline(
+        name, metadata, alias=alias, destination=destination, env=env, version=version
+    )
+
+    as_alias = f" as `{alias}`" if alias else ""
+    message = f"Micro-package `{name}` packaged{as_alias}! Location: {result_path}"
+    click.secho(message, fg="green")
+
+
+def _package_pipeline(  # pylint: disable=too-many-arguments
+    pipeline_name: str,
+    metadata: ProjectMetadata,
+    alias: str = None,
+    destination: str = None,
+    env: str = None,
+    version: str = None,
+) -> Path:
+    package_dir = metadata.source_dir / metadata.package_name
+    env = env or "base"
+
+    artifacts_to_package = _get_pipeline_artifacts(
+        metadata, pipeline_name=pipeline_name, env=env
+    )
+    # as the wheel file will only contain parameters, we aren't listing other
+    # config files not to confuse users and avoid useless file copies
+    configs_to_package = _find_config_files(
+        artifacts_to_package.pipeline_conf,
+        [f"parameters*/**/{pipeline_name}.yml", f"parameters*/**/{pipeline_name}/**/*"],
+    )
+
+    source_paths = (
+        artifacts_to_package.pipeline_dir,
+        artifacts_to_package.pipeline_tests,
+        configs_to_package,
+    )
+
+    # Check that pipeline directory exists and not empty
+    _validate_dir(artifacts_to_package.pipeline_dir)
+
+    destination = Path(destination) if destination else metadata.project_path / "dist"
```

Comment:
Worth mentioning in the release notes that the difference between `micropkg` and `pipeline` is that the former saves packages to `project / dist ` and the latter in `project / src / dist`, so that users are aware that they are not 100% equivalent.